add-on/chrome/bootstrap.js

```diff
@@ -485,6 +488,9 @@ var WindowListener = {
           // metadata about the page is available when this event fires.
           gBrowser.addEventListener("DOMTitleChanged", this);
           this._browserSharePaused = false;
+
+          this._mousemoveHandler = this.handleMousemove.bind(this);
```

Comment:
Instead of binding, you could do something similar to line 489: `gBrowser.addEventListener("mousemove", this);` where the `mousemove` event will be passed to `this.handleEvent`, which already has a `switch (event.type)` to dispatch appropriately -- in this case, dispatch to `this.handleMousemove`.